#!/usr/bin/env python
PACKAGE = "laserscan_multi_merger"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angle_min", double_t, 0, "Minimum angle of the output scan", -3.14, -3.14, 3.14)
gen.add("angle_max", double_t, 0, "Maximum angle of the output scan", 3.14, -3.14, 3.14)
gen.add("angle_increment", double_t, 0, "Angle increment of the output scan", 0.005759587, 0, 3.14)
gen.add("time_increment", double_t, 0, "Time increment of the output scan", 0.0, 0, 1)
gen.add("scan_time", double_t, 0, "Scan time of the output scan", 0.066666667, 0, 1)
gen.add("range_min", double_t, 0, "Range min of the output scan", 0.05, 0, 50)
gen.add("range_max", double_t, 0, "Range max of the output scan", 25.0, 0, 50)

exit(gen.generate(PACKAGE, "laserscan_multi_merger", "laserscan_multi_merger"))
